cloudferrylib/utils/cache.py

```diff
@@ -30,7 +30,9 @@ def __init__(self, func):
         self.func = func
         self.cache = {}
 
-    def __call__(self, *args):
+    def __call__(self, *args, **kwargs):
+        if kwargs:
+            return self.func(*args, **kwargs)
```

Comment:
This means cache will *never* be used for methods with key-value arguments used, please consider modifying the logic.